README.md

```diff
@@ -13,22 +11,11 @@ A minimal Angular starter for Universal JavaScript using TypeScript and Webpack
 
 [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
 
+## Getting Started
+There are two projects hosting in this reposity: `/cli` and `/custom-webpack`, choose the one that best fits your situation
```

Comment:
can you either list pros/cons of each one because it's not clear what situation fits who. Someone who is new to universal and maybe even Angular won't know what to do still. Someone starting out with Universal may not know either and will probably go for the CLI folder

Reply:
done